src/output/output.cpp

```diff
@@ -184,6 +231,29 @@ void Output::ForceWriteVtk(DataBlock &data) {
   idfx::popRegion();
 }
 
+#ifdef USE_HDF5
```

Comment:
to be consistent with MPI, this should probably be `#ifdef WITH_HDF5`, and same for all of the occurences of this flag.